apps/emqx/src/emqx_schema.erl

```diff
@@ -2418,21 +2418,21 @@ mk_duration(Desc, OverrideMeta) ->
 to_duration(Str) ->
     case hocon_postprocess:duration(Str) of
         I when is_integer(I) -> {ok, I};
-        _ -> {error, Str}
```

Comment:
hocon 0.39.9 has fixed a issue: `typerelf:from_string` conversion is not executed when value is integer  https://github.com/emqx/hocon/pull/259
In order to be compatible with the previous format.
```
%% {error, "1"} = hocon_postprocess:duration("1")
%% we should ensure:
to_duration("12") -> 12.
to_duration_s("12") -> 12s
to_duration_ms("12") -> 12ms

to_timeout_duration("12") -> 12
to_timeout_duration_ms("12") ->  12ms
to_timeout_duration_s("12") -> 12s
```